debian/rules

```diff
@@ -43,6 +43,19 @@ ifeq ($(DEB_HOST_ARCH_OS), linux)
 	endif
 endif
 
+# EOS-specific configuration
+GS_CONFIGURE_FLAGS += \
+	-Deos_updater=true \
+	-Dexternal_appstream=true \
+	-Dfwupd=false \
+	-Dgudev=false \
+	-Dhardcoded_popular=false \
+	-Dmalcontent=true \
+	-Dmogwai=true \
+	-Dpackagekit=false \
+	-Dsoup2=true \
+	-Dtests=true
```

Comment:
and this will override d922fd66794c4b318c2965db62f0942ea77f2a9b on OBS and presumably again fail the build.

Reply:
In theory tests are disabled a few lines below:

```
# DISABLED
override_dh_auto_test:
```

But removed nonetheless